components/validation/validation-custom-mixin.js

```diff
@@ -11,60 +11,49 @@ export const ValidationCustomMixin = superclass => class extends superclass {
 
 	constructor() {
 		super();
-		this._forElement = null;
+
+		this._eventSubscriberController = new EventSubscriberController(this, {},
+			{ eventName: 'd2l-validation-custom-connected' }
+		);
```

Comment:
Is the event subscriber needed? Seems like the custom validator isn't expecting descendent elements to fire this event.

Reply:
As far as I can tell, the event is used by `FormMixin` and `FormElementMixin` to allow for someone to put a `d2l-validation-custom` element _inside_ of one of them without adding a `for` id.  Like, if `d2l-input-text` wanted to instead just put a `d2l-validation-custom` element inside of itself.  There are no examples of this on the demo pages and I had to hack something together to see it happen.  When we actually implement this, I think it's worth trying to figure out if anyone actually ever uses this, and decide if that's something we'd still want to support.

Reply:
Whoa! Yeah I'd definitely lean towards just removing that functionality altogether.